integrated_channels/blackboard/client.py

```diff
@@ -40,15 +45,235 @@ def __init__(self, enterprise_configuration):
         self.config = apps.get_app_config('blackboard')
         self.session = None
         self.expires_at = None
+        self.create_course_url = "{}/learn/api/public/v3/courses".format(
```

Comment:
why not use the `COURSE_PATH` var here (and admittedly update it to be v3 instead of v1)?

Reply:
and I have a bug here! should have tested locally, my url is not complete ... will fix and be back

Reply:
and I can't use the existing vars due to the way they are setup..

Reply:
so they really don't have their get all courses endpoint on v3, but they do have their get/delete individual course on v3?

Reply:
why isn't this a constant with the other paths?